test-apps/appui-test-app/standalone/src/frontend/index.tsx

```diff
@@ -380,6 +381,25 @@ export class SampleAppIModelApp {
             section: StatusBarSection.Right,
             content: <AppLanguageSelect />,
           }),
+          StatusBarItemUtilities.createCustomItem({
```

Comment:
Could you add these to the storybook instead, i.e. see `MessageCenterField.stories.tsx`